class AlreadyPresentError(Exception):
    pass

class FileError(Exception):
    pass